qemu/qemu.go

```diff
@@ -2568,6 +2571,13 @@ func (config *Config) appendLogFile() {
 	}
 }
 
+func (config *Config) appendFwCfg() {
+	if config.FwCfg != "" {
```

Comment:
you may want to implement `Valid` and `QemuParams` methods for the `FwCfg struct` and use them here

Reply:
Implemented interface